8.0.0RC5-Dockerfile-block-1

```diff
@@ -0,0 +1,2 @@
+    && docker-php-ext-configure gd -enable-gd --with-freetype --with-jpeg --with-webp \
+    && pecl install xmlrpc \
```

Comment:
Why does PHP 8 need `pecl` instead of `docker-php-ext-install`?

Reply:
cause xmlrpc it got moved to pecl https://php.watch/versions/8.0/xmlrpc#:~:text=PHP%208.0%3A%20XMLRPC%20extension%20is%20moved%20to%20PECL&text=XMLRPC%20is%20an%20extension%20that,not%20maintained%20for%20several%20years.